### PR TITLE
updated JECs to Summer16_23Sep2016*V4, corrected bug in T1 MET, fixed small additional bug

### PR DESCRIPTION
**Pull request by oiorio**
Here's the update with new JECs and the bugfix, checked out the  @dmajumder @alefisico @decosa

### CODE REVIEW

File: test/b2gedmntuples_cfg.py [hunk 1]
```diff
@@ -654,14 +650,15 @@
 process.skimmedPatMET = cms.EDFilter(
     "PATMETSelector",
     #    src = cms.InputTag(metLabel, "", "PAT"),
-    src = cms.InputTag(metLabel, "", metProcess),
+#    src = cms.InputTag(metLabel, "", metProcess),
+    src = cms.InputTag(metLabel),
```
Comment (dmajumder) [inline test/b2gedmntuples_cfg.py:654]: if the met process is the same, we should drop the corresponding variable

File: test/b2gedmntuples_cfg.py [hunk 2]
```diff
@@ -654,14 +650,15 @@
...
     cut = cms.string("")
     )
 
 process.skimmedPatPuppiMET = cms.EDFilter(
     "PATMETSelector",
     #    src = cms.InputTag(metLabel, "", "PAT"),
-    src = cms.InputTag(puppimetLabel, "", metProcess),
+    src = cms.InputTag(puppimetLabel),
```
Comment (dmajumder) [inline test/b2gedmntuples_cfg.py:661]: if the met process is the same, we should drop the corresponding variable

Comment (oiorio) [reply test/b2gedmntuples_cfg.py:661]: We need the process to be the current one to save the T1 met which is recalculated on-the-fly.

At what level you want to drop the variable?

Comment (oiorio) [reply test/b2gedmntuples_cfg.py:661]: If you refer to the "metProcess" variable, it's needed later on as for taking the met filters one needs to differentiate between the case of data and MC where one takes either the RECO or the PAT process for the metUserData.

Comment (dmajumder) [reply test/b2gedmntuples_cfg.py:661]: indeed, I was talking about the "metProcess". We need it to access the filters

**Pull request merged**